src/main/resources/soy/prism.soy

```diff
@@ -1,4 +1,4 @@
-{namespace Prism.Macros.Code}
+{namespace JiraPrism.Macros.Code}
```

Comment:
Was this namespace rename necessary?

Reply:
I am no JS expert, but the prism lib seemed to overwrite the Prism object leading to unknown properties. JiraPrism was just a random pick. Feel free to try wit the original namespace, for me it didn't work.